jupyter-config/cern-config/k8s.sh

```diff
@@ -0,0 +1,2 @@
+export SPARK_DEMO_OPTION_CLUSTER_NAME=k8s
```

Comment:
Same comment as for the analytix equivalent, see above.

Reply:
Same as above